test/go/iterative_verifier_test.go

```diff
@@ -1,477 +0,0 @@
-	t.InsertCompressedRowInDb(42, testhelpers.TestCompressedData1, t.Ferry.TargetDB)
-
-	result, err := t.verifier.VerifyOnce()
-	t.Require().NotNil(result)
-	t.Require().Nil(err)
-	t.Require().True(result.DataCorrect)
-	t.Require().Equal("", result.Message)
-}
-
-func (t *IterativeVerifierTestSuite) TestVerifyDifferentCompressedSameDecompressedDataOncePass() {
-	t.Require().NotEqual(testhelpers.TestCompressedData3, testhelpers.TestCompressedData4)
-
-	t.InsertCompressedRowInDb(43, testhelpers.TestCompressedData3, t.Ferry.SourceDB)
-	t.InsertCompressedRowInDb(43, testhelpers.TestCompressedData4, t.Ferry.TargetDB)
-
-	result, err := t.verifier.VerifyOnce()
-	t.Require().NotNil(result)
-	t.Require().Nil(err)
-	t.Require().True(result.DataCorrect)
-	t.Require().Equal("", result.Message)
-}
-
-func (t *IterativeVerifierTestSuite) TestBeforeCutoverFailuresFailAgainDuringCutover() {
-	t.InsertRowInDb(42, "foo", t.Ferry.SourceDB)
-	t.InsertRowInDb(42, "bar", t.Ferry.TargetDB)
-
-	err := t.verifier.VerifyBeforeCutover()
-	t.Require().Nil(err)
-
-	result, err := t.verifier.VerifyDuringCutover()
-	t.Require().Nil(err)
-	t.Require().False(result.DataCorrect)
-	t.Require().Equal("verification failed on table: gftest.test_table_1 for pks: 42", result.Message)
-}
-
-func (t *IterativeVerifierTestSuite) TestBeforeCutoverCompressionFailuresFailAgainDuringCutover() {
-	t.InsertCompressedRowInDb(42, testhelpers.TestCompressedData1, t.Ferry.SourceDB)
-	t.InsertCompressedRowInDb(42, testhelpers.TestCompressedData2, t.Ferry.TargetDB)
-
-	err := t.verifier.VerifyBeforeCutover()
-	t.Require().Nil(err)
-
-	result, err := t.verifier.VerifyDuringCutover()
-	t.Require().Nil(err)
-	t.Require().False(result.DataCorrect)
-	t.Require().Equal(fmt.Sprintf("verification failed on table: %s.%s for pks: %s", "gftest", testhelpers.TestCompressedTable1Name, "42"), result.Message)
-}
-
-func (t *IterativeVerifierTestSuite) TestBeforeCutoverDifferentCompressedSameDecompressedDataPassDuringCutover() {
-	t.Require().NotEqual(testhelpers.TestCompressedData3, testhelpers.TestCompressedData4)
-
-	t.InsertCompressedRowInDb(43, testhelpers.TestCompressedData3, t.Ferry.SourceDB)
-	t.InsertCompressedRowInDb(43, testhelpers.TestCompressedData4, t.Ferry.TargetDB)
-
-	err := t.verifier.VerifyBeforeCutover()
-	t.Require().Nil(err)
-
-	result, err := t.verifier.VerifyDuringCutover()
-	t.Require().Nil(err)
-	t.Require().True(result.DataCorrect)
-	t.Require().Equal("", result.Message)
-}
-
-func (t *IterativeVerifierTestSuite) TestErrorsIfMaxDowntimeIsSurpassed() {
-	t.InsertRowInDb(42, "foo", t.Ferry.SourceDB)
-	t.InsertRowInDb(42, "bar", t.Ferry.TargetDB)
-
-	t.verifier.MaxExpectedDowntime = 1 * time.Nanosecond
-	err := t.verifier.VerifyBeforeCutover()
-	t.Require().Regexp("cutover stage verification will not complete within max downtime duration \\(took .*\\)", err.Error())
-}
-
-func (t *IterativeVerifierTestSuite) TestBeforeCutoverFailuresPassDuringCutover() {
```

Comment:
Again, the variations of VerifyBeforeCutover and VerifyDuringCutover is just covered by the main integration tests.